### ISSUE TITLE
train with UCF101

### ISSUE THREAD

**Issue by Atrin4000** [kenshohara/3D-ResNets-PyTorch#152]
can I use only UCF101 to train, and not going through the other ones like kinetic and activitynet?

**Comment by buaa-luzhi** [comment]
@Atrin4000 
I agree with you.
What is the current classification accuracy rate on the UC101 dataset?
Thanks

**Comment by slighting666** [comment]
@buaa-luzhi  
@Atrin4000 
Hi, I tested UCF101 with the resnet18 pre-training model provided by the author, but the effect was not very good. I wonder if it is because my hardware can't meet the requirements. What are your batch_size and epoch?
And then are you accurate?

**Comment by slighting666** [comment]
There is no resnet34,50 in the link
Do you have a pretraining model tuned in ucf101 or hmdb51? I need it very much, thank you. @Atrin4000 @buaa-luzhi
